### PR TITLE
Greene Manor

### PR DESCRIPTION
**Pull request by vtange**
Gina Green -> Gina Greene since she owns this place.

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -3966,67 +3966,67 @@ QUEST_LV_0200_20150323_003965	Are you the Revelator of the Goddesses or the one
...
-QUEST_LV_0200_20150323_003983	{memo X}Until the master comes back, I am going to do some base work for the large scale purification.{nl}Can you collect the five bottles of Tanus fluids from the Kvie farm?
-QUEST_LV_0200_20150323_003984	Thanks for helping me.{nl}Anyways, the girl who left the trees.. what is her identity?
-QUEST_LV_0200_20150323_003985	These trees are so mysterious.{nl}It's like the Revelator brought the clues of purification.
-QUEST_LV_0200_20150323_003986	You've done enough.{nl}I should tell the Druid Master that you've been a great help.{nl}
-QUEST_LV_0200_20150323_003987	Ah, I have one more thing to request to you lastly.
-QUEST_LV_0200_20150323_003988	This is the box that we discovered with the trees.{nl}If you get a chance to go to Shaton farm, please hand it over to Martineck.
-QUEST_LV_0200_20150323_003989	He likes to interfere with others' lives so he may have involved in some kind of matter.{nl}Maybe, we should ask the other people about the whereabout of him.
+QUEST_LV_0200_20150323_003982	$Oh my goddess, please look at this. The ritual was successful.{nl}The master would purify everything at once with trees like this one.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3982]: goddess -> Goddess

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -4045,7 +4045,7 @@ QUEST_LV_0200_20150323_004044	Well done. Well done.{nl}The Druid that you are lo
 QUEST_LV_0200_20150323_004045	He said lots of things about the labor conditions of the farmers so I moved him out with some excuse.{nl}After, I told him that the farmers at the workspace and the brewery are not returning, he disappeared fast.{nl}
 QUEST_LV_0200_20150323_004046	Well, it's not a lie so that shouldn't be a problem.{nl}Go to Neshiba intersections.
 QUEST_LV_0200_20150323_004047	I was acting kindly to Druid because of the farm located at the next side.{nl}But, if you annoy me, remember that you will be expelled from here.
-QUEST_LV_0200_20150323_004048	Druid Martineck
+QUEST_LV_0200_20150323_004048	Druid Martineque
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4048]: The name can be interpeted in many ways, so I don't know the 'official' name, but Martinek is also an option.

Comment (vtange) [inline QUEST_LV_0200.tsv:4048]: Many quest lines already say Martineque so let's leave it at that and change it once i see who he/she really is at Shaton Farm

**Comment by vtange** [comment]
should be good to merge

**Comment by Sunaries** [comment]
Good to merge. 

@vtange Check Discord.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**